tools/check_vlasiator_cfg.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-mpirun_cmd=$(which aprun > /dev/null && echo "aprun" || echo "mpirun")
+mpirun_cmd=$(which aprun &> /dev/null && echo "aprun" || echo "mpirun")
```

Comment:
Ah good addition of `&`! Now many times I drop `mpirun` even as e.g. on frontends or my laptop the direct serial running is sufficient to get the output. So should we change this line altogether, if we're fiddling with it anyway?

Reply:
Probably yes. I'm pretty sure `check_vlasiator_cfg` is broken on mpirun as well, I've just run it serially since it's just reading the parameters.